pack/mts_encounter.json

```
@@ -1965,6 +1967,7 @@
 		"quantity": 2,
 		"set_code": "enchantress",
 		"set_position": 4,
+		"text": "Attach to your identity. You cannot make basic attacks or play attack events. Alter-Ego Action: spend [[energy]] [[mental]] resources -> discard this card",
```

With missing formatting:

```suggestion
		"text": "Attach to your identity.\nYou cannot make basic attacks or play [[Attack]] events.\n<b>Alter-Ego Action</b>: Spend [energy] [mental] resources → discard this card.",
```